### ISSUE TITLE
interface conversion: interface {} is *schema.Set, not []interface {}

### ISSUE THREAD

**Issue by mnaser** [vexxhost/terraform-provider-uptimerobot#3]
Seems like the most recent change that merged broke things

Not sure if you didn't run into that locally @johanfleury

Also, noticed you're a Montreal local too! 👋

**Comment by mnaser** [comment]
The failing build is here:

https://github.com/vexxhost/terraform-provider-uptimerobot/runs/4120019095?check_suite_focus=true

Traceback:

```
panic: interface conversion: interface {} is *schema.Set, not []interface {}
23

24
goroutine 1185 [running]:
25
github.com/vexxhost/terraform-provider-uptimerobot/internal/provider.resourceMonitorCreate(0xd9bc80, {0xe74f80, 0xc0002458a0})
26
	/home/runner/work/terraform-provider-uptimerobot/terraform-provider-uptimerobot/internal/provider/resource_uptimerobot_monitor.go:149 +0xb15
27
github.com/hashicorp/terraform-plugin-sdk/helper/schema.(*Resource).Apply(0xc000108c00, 0xc000a901e0, 0xc000213740, {0xe74f80, 0xc0002458a0})
28
	/home/runner/go/pkg/mod/github.com/hashicorp/terraform-plugin-sdk@v1.0.0/helper/schema/resource.go:305 +0x438
29
github.com/hashicorp/terraform-plugin-sdk/helper/schema.(*Provider).Apply(0xc000108d00, 0xc0008f18d8, 0xea992d, 0xf)
30
	/home/runner/go/pkg/mod/github.com/hashicorp/terraform-plugin-sdk@v1.0.0/helper/schema/provider.go:289 +0x70
31
github.com/hashicorp/terraform-plugin-sdk/internal/helper/plugin.(*GRPCProviderServer).ApplyResourceChange(0xc0000b5818, {0xc000a32540, 0x5035a6}, 0xc000a32540)
32
	/home/runner/go/pkg/mod/github.com/hashicorp/terraform-plugin-sdk@v1.0.0/internal/helper/plugin/grpc_provider.go:885 +0x7a5
33
github.com/hashicorp/terraform-plugin-sdk/internal/tfplugin5._Provider_ApplyResourceChange_Handler({0xe72d00, 0xc0000b5818}, {0x1040ef0, 0xc00001da70}, 0xc000a32480, 0x0)
34
	/home/runner/go/pkg/mod/github.com/hashicorp/terraform-plugin-sdk@v1.0.0/internal/tfplugin5/tfplugin5.pb.go:3189 +0x170
35
google.golang.org/grpc.(*Server).processUnaryRPC(0xc0000e4c60, {0x1050938, 0xc0007c6000}, 0xc00028b000, 0xc0009c1c80, 0x16503e0, 0x0)
36
	/home/runner/go/pkg/mod/google.golang.org/grpc@v1.23.0/server.go:995 +0xe0d
37
google.golang.org/grpc.(*Server).handleStream(0xc0000e4c60, {0x1050938, 0xc0007c6000}, 0xc00028b000, 0x0)
38
	/home/runner/go/pkg/mod/google.golang.org/grpc@v1.23.0/server.go:1275 +0xa56
39
google.golang.org/grpc.(*Server).serveStreams.func1.1()
40
	/home/runner/go/pkg/mod/google.golang.org/grpc@v1.23.0/server.go:710 +0x98
41
created by google.golang.org/grpc.(*Server).serveStreams.func1
42
	/home/runner/go/pkg/mod/google.golang.org/grpc@v1.23.0/server.go:708 +0xef
43
FAIL	github.com/vexxhost/terraform-provider-uptimerobot/internal/provider	3.758s
44
FAIL
```

**Comment by mnaser** [comment]
I've got a fix for this, pushing now..

**Comment by johanfleury** [comment]
I’m sorry @mnaser , I should’ve opened this PR as a draft, I wasn’t expecting you to merge right away (I thought acceptance tests would run before).

I made a fix in #4, didn’t see this issue before making the PR.

**Comment by mnaser** [comment]
Closed in https://github.com/vexxhost/terraform-provider-uptimerobot/commit/b062d363989d42ee6be3beec81d52f91c5a68f26 and also the rate limits issue has been addressed.
